MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/ComposeContentViewModel.swift

```diff
@@ -228,34 +228,23 @@ public final class ComposeContentViewModel: NSObject, ObservableObject {
         }
 
         // set limit
-        let _configuration: Mastodon.Entity.Instance.Configuration? = {
+        if let configuration: Mastodon.Entity.V2.Instance.Configuration = {
+            var configuration: Mastodon.Entity.V2.Instance.Configuration? = nil
+            context.managedObjectContext.performAndWait {
+                let authentication = authContext.mastodonAuthenticationBox.authentication
+                configuration = authentication.instance(in: context.managedObjectContext)?.configurationV2
+            }
+            return configuration
+        }() { applyInstanceConfiguration(configuration) }
```

Comment:
I really, really dislike this `let ... = { }()`-syntax. So much visual noise.

No need to change it, I just wanted to complain :D

Reply:
It would be so cool if Swift supported the Kotlin-esque `?.let {}` here because I think this would be cool and you'd like it, hm now that I'm writing this I'Ve got an idea, let me quickly check